src/kudu/consensus/leader_election.h

```diff
@@ -434,6 +434,7 @@ class LeaderElection : public RefCountedThreadSafe<LeaderElection> {
                  VoteRequestPB request,
                  gscoped_ptr<VoteCounter> vote_counter,
                  MonoDelta timeout,
+                 bool enforce_rpc_token,
```

Comment:
why is leader election stating to have concepts like enforce_rpc_token in its interface.
We seem to be leaking very low level concepts into a higher level interface.

Reply:
This has to do with who is initiating the rpc request. Currently the way it's done is that it's set by the initiator. Leader election is at the same level as RaftConsensus; they both are the initiator of the RPC, which is why this is here.

I was struggling a bit with this mostly because the actual best place to do it it probably in ConsensusService, but it was hard to get the option into there.

That can be re-examined.